### PR TITLE
Add ADAC

### PR DESCRIPTION
**Pull request by kmpoppe**
Similarweb Ranking 2,905
Category set to Insurance as they _also_ offer different insurances around car ownership but also general insurances, even though they started with and are first and foremost an Automobile Club that offer Roadside Assistance.

### CODE REVIEW

File: img/a/adac.de.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...p://www.w3.org/2000/svg" width="608.9" height="609.7" version="1.0"><g font-family="fnt0,&quot;Futura&quot;,Arial" font-size="3.153" letter-spacing="-.063" word-spacing="0"><path fill="#ffcf00" d="M0 0h608.9v609.7H0V0z" class="ps0318 ps161 ps20"/><path fill="#231f1f" d="M29.28 216.604h36.906l7.822-38.106h41.113l6.82 38.106h45.325L129.563 32.089H70.798L29.28 216.604zM95.468 70.196h.4l13.037 76.212h-28.68l15.243-76.212" class="ps010 ps161 ps20"/><path fill="#231e1f" d="M179.502 216.604h55.155c55.351 0 76.814-44.123 76.814-96.269 0-54.15-26.275-88.246-82.43-88.246h-49.539v184.515zM225.23 64.179h3.81c30.283 0 33.292 34.095 33.292 54.151 0 24.067 0 64.18-32.09 64.18h-5.012V64.178M306.055 216.604h38.107l8.021-38.106h42.318l7.221 38.106h46.73L409.342 32.089h-60.37l-42.918 184.515zm68.188-146.408h.604l13.237 76.212H358.6l15.643-76.212M538.102 96.268c0-12.033-3.21-34.095-19.654-34.095-19.253 0-22.063 28.078-22.063 62.174 0 38.106 1.805 60.168 23.064 60.168 16.848 0 20.659-14.04 21.66-36.101l39.511 4.011c0 40.112-23.665 66.185-63.176 66.185-55.355 0-70.398-46.13-70.398-94.263 0-46.129 14.843-96.269 70.398-96.269 43.722 0 63.176 30.084 63.176 68.19h-42.518" class="ps07 ps162 ps20"/></g></svg>
```
Comment (hkamran80) [inline img/a/adac.de.svg]: Here's a minified version.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 609 610"><path fill="#ffcf00" d="M0 0h609v610H0Z"/><path fill="#231f1f" d="M29 217h37l8-39h41l7 39h45L130 32H71ZM95 70h1l13 76H80l15-76"/><path fill="#231e1f" d="M180 217h55c55 0 76-45 76-97 0-54-26-88-82-88h-49Zm45-153h4c30 0 33 34 33 54 0 24 0 65-32 65h-5V64m81 153h38l8-39h43l7 39h46L409 32h-60l-43 185Zm68-147h1l13 76h-29l15-76m164 26c0-12-3-34-20-34-19 0-22 28-22 62 0 38 2 61 23 61 17 0 21-15 22-37l40 4c0 41-24 67-64 67-55 0-70-47-70-95 0-46 15-96 70-96 44 0 64 30 64 68h-43"/></svg>
```

File: entries/a/adac.de.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "Raisin": {
+    "domain": "adac.de",
+    "tfa": [
+      "sms",
+      "call"
+    ],
```
Comment (hkamran80) [inline entries/a/adac.de.json:7]: Please attach some screenshots of the 2FA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.

File: entries/a/adac.de.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "Raisin": {
```
Comment (hkamran80) [inline entries/a/adac.de.json]: Shouldn't this be "ADAC"?

```suggestion
  "ADAC": {
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by kmpoppe** [comment]
It seems that 2fa isn't mandatory to login and to see personal data, but only for specific parts of the members only area ... which is hazardous bullocks, alas, they don't qualify for being listed.

**Comment by hkamran80** [comment]
Do you want to list it as not supporting 2FA then?

**Comment by kmpoppe** [comment]
I will write to their IT department and ask what happened while planning that and see if there's a change planned in the not too distant future.

So, no change in the close state of this PR for now.

**Pull request closed without merging**